hyper/tls.py

```diff
@@ -94,9 +96,17 @@ def init_context(cert_path=None, cert=None, cert_password=None):
         encrypted and no password is needed.
     :returns: An ``SSLContext`` correctly set up for HTTP/2.
     """
+    cafile = cert_path or cert_loc
+    if not cafile or not path.exists(cafile):
+        errMsg = ("No certificate found at " + str(cafile) + ". Either " +
+                  "ensure the default cert.pem file is included in the " +
+                  "distribution or provide a custom certificate when " +
+                  "creating the connection.")
+        raise Exception(errMsg)
```

Comment:
Please don't raise plain `Exception`s: define a custom exception in our `exceptions.py` file and raise that. =)